Stardaze/OutputFormatter.swift

```diff
@@ -221,7 +338,11 @@ internal struct PrettyPrintedStringFormatter: Visitor {
     internal func visit(_ fragments: [Fragment]) -> String {
         var finishedString = ""
 
-        for fragment in fragments {
+        for (index, fragment) in zip((0..<fragments.count), fragments) {
```

Comment:
- [ ] Treat `..<` like any other operator and add a single space on either side of it